### DISCUSSION TITLE
Queue / Playlist / Autoplay Function

### DISCUSSION THREAD

**Discussion by lemonventures** [f1multiviewer/issue-tracker#394] [Feature requests]
Would love a playlist/queue feature for past sessions, where you can create your desired watchlist, hit play and just let it roll. 

E.g. I'm trying to watch the 1984 season highlights from the archive and want each 10 minute clip to play one after the next, rather than having to manually go back and select the next race highlight every few minutes. 

F1TV's autoplay feature is a nightmare, throwing you from whatever you've just watched into a seemingly random selection for the next video.

**Comment by gramathy** [comment]
I would also like to request a playlist feature where you can set up either a manual or automatically generated playlist using categories (e.g. a checkbox list "include practice" "include analysis" and boxes for junior series) would be fantastic, I tend to exercise with races on and the junior series are great for padding a session but I hate having to stop and get up after a sprint race.
